gnuradio-runtime/lib/qa_circular_file.cc

```diff
@@ -24,9 +24,8 @@
 #include "config.h"
 #endif
 
-#include "qa_circular_file.h"
 #include "circular_file.h"
-#include <cppunit/TestAssert.h>
+#include <boost/test/unit_test.hpp>
 #include <iostream>
 #include <stdio.h>
```

Comment:
... we use neither iostreams nor any stdio.h things, I think?

Reply:
Nope. Gone.